### PR TITLE
Correct Darcy velocity calculation in GroundwaterFlow process

### PR DESCRIPTION
**Pull request by TomFischer**
Fixes wrong access to hydraulic conductivity in calculation of darcy velocity.

### CODE REVIEW

File: ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h [hunk 1]
```diff
@@ -155,7 +155,8 @@ class LocalAssemblerData : public GroundwaterFlowLocalAssemblerInterface
         // fetch hydraulic conductivity
         ParameterLib::SpatialPosition pos;
         pos.setElementID(_element.getID());
-        auto const k = _process_data.hydraulic_conductivity(t, pos)[0];
+        auto const k = hydraulicConductivity<GlobalDim>(
```
Comment (wenqing) [inline ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h]: You can use MPL and formEigenTensor. With that the function of hydraulicConductivity can be removed.

Comment (TomFischer) [reply ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h]: PR #2820 will change it.

**Comment by wenqing** [comment]
There are already changes in PR #2820. What it the purpose of this PR?

File: ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h (hunk 1 above)
Comment (wenqing) [reply ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h]: I see this might be a quick fixing.

**Pull request merged**